src/main/kotlin/racingcar/ResultView.kt

```diff
@@ -0,0 +1,10 @@
+package racingcar
+
+const val RESULT_MESSAGE = "실행 결과"
+
+fun printGameResult(gameResults: List<GameResult>) {
+    println(RESULT_MESSAGE)
+    for (gameResult in gameResults) {
+        println(gameResult)
+    }
```

Comment:
아래와 같이 변경해볼 수도 있을 것 같아요. 😃
```suggestion
    gameResults.forEach { println(it) }
```